DESCRIPTION

```diff
@@ -45,6 +45,8 @@ Suggests:
     pkgdown (>= 1.1.0),
     rmarkdown,
     roxygen2,
+    RcppEigen,
+    RcppArmadillo,
```

Comment:
Why does usethis itself need these in Suggests?

Reply:
@jennybc the `Suggests:` change was to allow the unit tests for `use_rcpp_armadillo()` and `use_rcpp_eigen()` to work on Travis-CI. Initially, these tests were skipped via `skip_if_not_installed()` as it would add onto the build requirements. However, after being requested by @hadley, the packages were added to `Suggests:` with the aforementioned disclaimers after each code review. I'm more than happy to remove them and add `skip_if_not_installed()`. 

c.f. https://github.com/r-lib/usethis/pull/601/files/5b0d7911cdadb396cf2c8e831ca501aa053f52a5#diff-695f4452520470c8e82fa2f184ee14b1